source/whats-new.txt

```diff
@@ -12,9 +12,20 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.1:
+
+What's New in 4.1
+-----------------
+
+New features of the 4.1 Node.js driver release include:
+
+- Added support for load balancer to specify how a driver should behave when connected
```

Comment:
What does this provide the user? And is there a link we can provide after expanding this?

Reply:
I couldn't find any existing docs on load balancer, so I'm thinking to make that part of the ticket to document it